### ISSUE TITLE
Benefits of using this module over native tailwindcss?

### ISSUE THREAD

**Issue by sindrepm** [nuxt-modules/tailwindcss#449]
Hi,

I've used this module in the past, but now that I'm starting a new project I'm wondering are there any reasons to use this module instead of just following the official tailwind installation [instructions](https://tailwindcss.com/docs/guides/nuxtjs)? Is it just less configuration or any additional features/benefits?

**Comment by carbontwelve** [comment]
I couldn't get hot-reloading working with following the official tailwind installation ant Nuxt3; with using this module it just works.

**Comment by sindrepm** [comment]
@carbontwelve I was able to install the latest Tailwind version following the official instructions (on Nuxt 2) and everything seems to be working fine, including hot-reloading. I did run into an issue where it would infinitely try to reload/rebuild every second or so, but that was fixed by making a small change to `tailwind.config.js`:

```
module.exports = {
  content: [
    './components/**/*.{vue,js}',
    './layouts/**/*.vue',
    './pages/**/*.vue',
    './plugins/**/*.{js,ts}',
    './nuxt.config.js',
  ],
  ...
```
I had to replace `./nuxt.config.{js,ts}` with `./nuxt.config.js`.

I'm sure this module automates some of the configuration, but I prefer to be able to control which version of tailwind is used.
